modules/nw-multus-bridge-object.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+
+[id="nw-multus-bridge-object_{context}"]
+= Configuration for bridge
+
+// TODO - duplicated in ipvlan, copy changes from there to here.
+
+.bridge CNI plug-in YAML configuration
+[source,yaml]
+----
+name: <name> <1>
+namespace: <namespace> <2>
+rawCNIConfig: '' <3>
+type: Raw
+----
+<1> `name`: Specify the name of the `NetworkAttachmentDefinition` created from
+the `rawCNIConfig` JSON object.
+
+<2> `namespace`: Specify the namespace to create the network attachment in. If
+a value is not specified, the `default` namespace is used.
+
+<3> `rawCNIConfig`: Specify the CNI plug-in configuration.
+
+.bridge CNI plug-in JSON configuration object
+[source,json]
+----
+{
+  "cniVersion": "0.3.1",
+  "name": "<name>", <1>
+  "type": "bridge",
+  "bridge": "<bridge>", <2>
+  "ipam": { <3>
+    ...
+  },
+  "ipMasq": false, <4>
+  "isGateway": false, <5>
+  "isDefaultGateway": false, <6>
+  "forceAddress": false, <7>
+  "hairpinMode": false, <8>
+  "promiscMode": false, <9>
+  "vlan": <vlan>, <10>
+  "mtu": <mtu> <11>
+}
+----
+<1> `name`: Specify the name of the `NetworkAttachmentDefinition`.
+
+<2> `bridge`: Specify the name of the virtual bridge to use. If the bridge
+interface does not exist on the host, it is created. The default value is
+`cni0`.
+
+<3> `ipam`: Specify a configuration object for the ipam CNI plug-in. The plug-in
```

Comment:
@kalexand-rh, so the IPAM configuration is identical for all plug-ins, and thus lives in a separate module. So it may not necessarily be "following", although in my assemblies, it will. How to handle this implicit dependency between modules, if this module is to remain independent per mod-docs?